types/block.go

```diff
@@ -135,6 +135,15 @@ func (b *Block) HashesTo(hash []byte) bool {
 	return bytes.Equal(b.Hash(), hash)
 }
 
+// Size returns size of the block in bytes.
+func (b *Block) Size() int {
+	bz, err := cdc.MarshalBinaryBare(b)
```

Comment:
expensive to call for each block?

Reply:
Can we make a call on this? Do we know about the cost?

Reply:
![screen shot 2018-06-20 at 11 16 05](https://user-images.githubusercontent.com/1282182/41643102-615b08d4-747b-11e8-8ab8-122fb44143fa.png)

~10ms